melodic/distribution.yaml

```
@@ -2922,6 +2922,11 @@ repositories:
       url: https://github.com/PickNikRobotics/moveit_sim_controller.git
       version: melodic-devel
     status: maintained
+  moveit_tutorials:
+    doc:
+      type: git
+      url: https://github.com/ros-planning/moveit_tutorials.git
+      version: master
```

I believe the correct branch would be `melodic-devel`
```suggestion
      version: melodic-devel
```

---

Thanks. Accepted change. This should be ready for final review.